extensions/vscode/src/views/homeView.ts

```diff
@@ -354,7 +357,7 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
   private async refreshConfigurationData() {
     try {
       const api = await useApi();
-      const response = await api.configurations.getAll();
+      const response = await api.configurations.getAll({ recursive: true });
```

Comment:
recursively refreshing everytime.. We probably can figure out a more targeted approach, but lets do that in another PR as an optimization.